test/tests.yml

```diff
@@ -30,760 +47,872 @@
+    - name: aws_rockylinux_stable
 
-- scenario: cilium_containerd
-  initVersion: v1.28.3
+- scenario: upgrade_containerd_external
+  initVersion: v1.27.11
+  upgradedVersion: v1.28.7
   infrastructures:
-    - name: azure_default
-    - name: azure_centos
-    - name: azure_flatcar
-    - name: azure_rhel
-    - name: azure_rockylinux
+    - name: aws_amzn_stable
+    - name: aws_centos_stable
+    - name: aws_default_stable
+    - name: aws_flatcar_stable
+    - name: aws_rhel_stable
+    - name: aws_rockylinux_stable
+    - name: azure_centos_stable
+    - name: azure_default_stable
+    - name: azure_flatcar_stable
+    - name: azure_rhel_stable
+    - name: azure_rockylinux_stable
+    - name: digitalocean_centos_stable
+    - name: digitalocean_default_stable
+    - name: digitalocean_rockylinux_stable
+    - name: equinixmetal_centos_stable
+    - name: equinixmetal_default_stable
+    - name: equinixmetal_flatcar_stable
+    - name: equinixmetal_rockylinux_stable
     - name: gce_default
+    - name: hetzner_centos_stable
+    - name: hetzner_default_stable
+    - name: hetzner_rockylinux_stable
+    - name: openstack_centos_stable
+    - name: openstack_default_stable
+    - name: openstack_flatcar_stable
+    - name: openstack_rhel_stable
+    - name: openstack_rockylinux_stable
+    - name: vsphere_centos_stable
+    - name: vsphere_default_stable
+    - name: vsphere_flatcar_stable
 
-- scenario: cilium_containerd
-  initVersion: v1.27.7
+- scenario: upgrade_containerd_external
+  initVersion: v1.28.7
+  upgradedVersion: v1.29.2
   infrastructures:
-    - name: azure_default
-    - name: azure_centos
-    - name: azure_flatcar
-    - name: azure_rhel
-    - name: azure_rockylinux
+    - name: aws_amzn_stable
+    - name: aws_centos_stable
+    - name: aws_default_stable
+    - name: aws_flatcar_stable
+    - name: aws_rhel_stable
+    - name: aws_rockylinux_stable
+    - name: azure_centos_stable
+    - name: azure_default_stable
+    - name: azure_flatcar_stable
+    - name: azure_rhel_stable
+    - name: azure_rockylinux_stable
+    - name: digitalocean_centos_stable
+    - name: digitalocean_default_stable
+    - name: digitalocean_rockylinux_stable
+    - name: equinixmetal_centos_stable
+    - name: equinixmetal_default_stable
+    - name: equinixmetal_flatcar_stable
+    - name: equinixmetal_rockylinux_stable
     - name: gce_default
+    - name: hetzner_centos_stable
+    - name: hetzner_default_stable
+    - name: hetzner_rockylinux_stable
+    - name: openstack_centos_stable
+    - name: openstack_default_stable
+    - name: openstack_flatcar_stable
+    - name: openstack_rhel_stable
+    - name: openstack_rockylinux_stable
+    - name: vsphere_centos_stable
+    - name: vsphere_default_stable
+    - name: vsphere_flatcar_stable
 
-- scenario: cilium_containerd_external
-  initVersion: v1.28.3
+###########################
+# ALTERNATIVE CNI PLUGINS #
+###########################
+
+- scenario: calico_containerd
```

Comment:
Let's remove `calico_containerd` scenario and only keep `calico_containerd_external` tests